modules/hcp-virt-create-hc-console.adoc

```diff
@@ -22,11 +22,17 @@ To create a hosted cluster with the KubeVirt platform by using the console, comp
 +
 [NOTE]
 ====
-* If you want to use predefined values to automatically populate fields in the console, you can create a {VirtProductName} credential. For more information, see _Creating a credential for an on-premises environment_.
+* If you want to use predefined values to automatically populate fields in the console, you can create a {VirtProductName} credential. For more information, see "Creating a credential for an on-premises environment".
 
 * On the *Cluster details* page, the pull secret is your {product-title} pull secret that you use to access {product-title} resources. If you selected a {VirtProductName} credential, the pull secret is automatically populated.
 ====
 
+. On the **Node pools** page, expand the **Networking options** section and configure the networking options for your node pool:
```

Comment:
For all of these, just one asterisk is needed to bold

```suggestion
. On the *Node pools* page, expand the *Networking options* section and configure the networking options for your node pool:
```

Reply:
Thanks! Will fix.